### DISCUSSION TITLE
Application for use has not been answered..

### DISCUSSION THREAD

**Discussion by liangliangtw** [community/community#16074] [Copilot Conversations] [answered]
After applying, it has been prompted to be on the waiting list, and no email has been received to enable it...
![image](https://user-images.githubusercontent.com/46476233/166862693-99b6048c-626f-467c-9d73-66da9084d71a.png)
It has been two or three months since the application.....

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
